lib/blocking-io.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function () {
+  if (process.stdout._handle && typeof process.stdout._handle.setBlocking === 'function') {
+    process.stdout._handle.setBlocking(true)
+    process.stderr._handle.setBlocking(true)
+  }
```

Comment:
Solution seems fine to me.

One note: If `_handle` exists it's always a `net.Socket` (or inherits from it), and thus `setBlocking()` should always exist.
(This has not been changed in years.)